.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+ python:
+    - "3.6"
+install:
+  - pip install -r requirements.txt
+  - nightly
```

Comment:
Remove this if you are not using it. Kindly check that the format is an exact match to this one with specificity to only what you need: https://docs.travis-ci.com/user/languages/python/